web/docs/operators/sinks/explore.md

```diff
@@ -0,0 +1,88 @@
+# explore
+
+Renders pipeline data in a terminal user interface (TUI).
+
+## Synopsis
+
+```
+explore [-f|--fullscreen <bool>] [-h|--height <int>] [-w|--width <int>]
+        [-n|--navigator-position <string>] [-N|--navigator-auto-hide]
+        [-T|--hide-types]
+```
+
+## Description
+
+The `explore` operator renders the data in the pipeline in a terminal user
+interface (TUI).
+
+Press `?` to get an help screen that shows available key bindings.
+
+Use the arrow keys or `j`/`k`/`h`/`l` to navigate through focusable components.
+
+You can also use the mouse to scroll vertically and horizontally by moving the
+pointer relative to the center of the TUI in a specific direction. For example,
+moving the point to the right from the center point scrolls to the right.
+
+The implementation uses the [FTXUI](https://arthursonzogni.github.io/FTXUI/)
+library for functional terminal interfaces in C++.
+
+### `-f|--fullscreen`
+
+Renders the TUI in fullscreen mode, as opposed to fitting it into the terminal.
+
+### `-h|--height <int>`
+
+The height of the TUI. Only actionable when `-w|--width` is also present.
+
+Defaults to 0, which means automatic detection.
+
+### `-w|--width <int>`
+
+The width of the TUI. Only actionable when `-h|--height` is also present.
+
+Defaults to 0, which means automatic detection.
+
+### `-n|--navigator-position <string>`
+
+The position of the navigator pane. Must be one of `left`, `right`, `top`,
+`bottom`.
```

Comment:
When this is set to `left` the schema name is truncated most of the time:

<img width="979" alt="image" src="https://github.com/tenzir/tenzir/assets/4488655/59a94a4f-5998-440a-a702-ea13797889a5">

Reply:
That's the same underlying issue as above. That's an artifact of the terminal being too narrow. The layout is that we have two "containers" and the downsizing that FTXUI does to fit the terminal eats away from both.

Reply:
To be addressed in a future PR.